index.html

```diff
@@ -38,5 +38,118 @@ <h2 class="text-container" id="sdsection">
+          <li class="tech-tag">css</li>
+          <li class="tech-tag">JavScript</li>
+          <li class="tech-tag">html</li>
+        </ul>
+        <br/>
+        <button class="see-project">See Project</button>
+      </div> <br>
+      <div class="card">
+        <div class="img-wrapper"><img class="img-placeholder" src="" /></div>
+        <h2 class="card-name">Multi-Post Stories<br>Gain+Glory</h2>
+        <ul class="technologies">
+          <li class="tech-tag">Ruby on rails</li>
+          <li class="tech-tag">css</li>
+          <li class="tech-tag">JavScript</li>
+          <li class="tech-tag">html</li>
+        </ul>
+        <br/>
+        <button class="see-project">See Project</button>
+      </div> <br>
+      <div class="card">
+        <div class="img-wrapper"><img class="img-placeholder" src="" /></div>
+        <h2 class="card-name">Multi-Post Stories<br>Gain+Glory</h2>
+        <ul class="technologies">
+          <li class="tech-tag">Ruby on rails</li>
+          <li class="tech-tag">css</li>
+          <li class="tech-tag">JavScript</li>
+          <li class="tech-tag">html</li>
+        </ul>
+        <br/>
+        <button class="see-project">See Project</button>
+      </div> <br>
+      <div class="card">
+        <div class="img-wrapper"><img class="img-placeholder" src="" /></div>
+        <h2 class="card-name">Multi-Post Stories<br>Gain+Glory</h2>
+        <ul class="technologies">
+          <li class="tech-tag">Ruby on rails</li>
+          <li class="tech-tag">css</li>
+          <li class="tech-tag">JavScript</li>
+          <li class="tech-tag">html</li>
+        </ul>
+        <br/>
+        <button class="see-project">See Project</button>
+      </div> <br>
+      <div class="card">
+        <div class="img-wrapper"><img class="img-placeholder" src="" /></div>
+        <h2 class="card-name">Multi-Post Stories<br>Gain+Glory</h2>
+        <ul class="technologies">
+          <li class="tech-tag">Ruby on rails</li>
+          <li class="tech-tag">css</li>
+          <li class="tech-tag">JavScript</li>
+          <li class="tech-tag">html</li>
+        </ul>
+        <br/>
+        <button class="see-project">See Project</button>
+      </div> <br>
+    <section class="about-me">
+      <h2 class="about-me-title">About me</h2>
+      <p class="about-me-description"> I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, 
+        don’t hestiate to contact me.</p>
+      <button class="resume-button">Get My Resume</button>
+    </section> <br>   
+    <div class="card-2">
+      <img class="icon-lang" src="icon-lang.png" />
+      <h2 class="card-name-2">Languages</h2>
+      <ul class="technologies-2">
+        <li class="tech-tag-2">JavaScript</li>
+        <li class="tech-tag-2">Ruby</li>
+        <li class="tech-tag-2">Html</li>
+        <li class="tech-tag-2">CSS</li>
+      </ul>
```

Comment:
- [ ] Please add the right sizes, margins, and padding to all your elements since this list of languages is overflowing past the card's border. Please remember to fix this issue in the Frameworks and Skills lists too.

  ![port72](https://user-images.githubusercontent.com/87394385/213252089-75279efb-9b4d-41c5-a970-7864cf3acb66.JPG)